.travis.yml

```diff
@@ -47,6 +47,20 @@ matrix:
            PROJECT_URL=https://github.com/joblib/joblib.git
       python: 3.7
       if: commit_message =~ /(\[ci downstream\]|\[ci joblib\])/
+    - os: linux
+      env: PROJECT=ray
+      python: 3.7
+      if: commit_message =~ /(\[ci downstream\]|\[ci ray\])/
+      install:
+        - pip install --upgrade -r dev-requirements.txt
+        - pip install setproctitle psutil ray==0.6.4
+        - PROJECT_DIR="$(python -c "import os,ray,sys;print(os.path.dirname(ray.__file__));sys.stdout.flush()")"
```

Comment:
Because this job is running python 3 only, we could directly write:

```bash
PROJECT_DIR="$(python -c "import os,ray; print(os.path.dirname(ray.__file__), flush=True)")"
```